index.Rmd

```diff
@@ -304,101 +303,127 @@ ggplot(us_data, aes(x = med_income, y = med_prop_val)) +
+Most of the data points from the 214 observations in this data set are bunched up in the lower left-handside of this visualization. This is because many many more people in Canada speak the two official languages (English and French). Thus to answer our question, we will need to adjust the scale of the x and y axes so that they are on a log scale. We can again add additional layers to the plot object using the `+` symbol to do this:
+
+```{r mother_tongue_vs_most_at_home_scale, fig.width=5.75, fig.height=5, warning=FALSE}
+ggplot(can_lang, aes(x = most_at_home, y = mother_tongue)) +
+  geom_point() +
+  xlab("Language spoken most at home (number of Canadian residents)") +
+  ylab("Mother tongue (number of Canadian residents)") +
+  scale_x_log10(labels = scales::comma) +
+  scale_y_log10(labels = scales::comma)
+```
+
+From this visualization we see that for the 214 languages in this data set, as the number of people who have a language as their mother tongue increases, so does the number of people who speak that language at home. When we see 
+two variables do this, we call this a *positive relationship*. Because the points are fairly close together, we can say that the relationship is strong. Because drawing a straight line through these 
+points would fit the pattern we observe quite well, we say that it's linear.
 
 Learning how to describe data visualizations is a very useful skill. We will provide descriptions for you in this course (as we did above) until we get to Chapter 4, 
 which focuses on data visualization. Then, we will explicitly teach you how to do this yourself, and how to not over-state or over-interpret the results 
 from a visualization.
 
-### Coloring points by group
-Now we'll move onto the second part of our exploratory data analysis question: when considering the relationship between median household income and median property 
-value, is there a pattern in which party each state voted for in the 2016 US election? One common way to explore this is to colour the data points on the 
-scatter plot we have already created by group/category. For example,
-given that we have the party each state voted for in the 2016 US Presidential election in the column named `party`, we can colour the points in our previous 
-scatter plot to represent who each stated voted for. 
-
-To do this we modify our scatter plot code above. Specifically, we will add an argument to the `aes` function, specifying that the points should be coloured by the `party` column:
+### Changing the units
 
-```{r relevel factors for colour, include=FALSE}
-us_data <- us_data %>% mutate(party = factor(party, levels = c("Republican", "Not Applicable", "Democratic")))
+```{r}
```

Comment:
I think this code block needs a little explanation before it

Reply:
I am not sure what this comment is referring to, can you please clarify @leem44 ?